### ISSUE TITLE
Logo alt text should not display filename

### ISSUE THREAD

**Issue by thijstriemstra** [fluidd-core/fluidd#1008]
### Fluidd Version

1.23

### Browser

Firefox

### Device

Desktop PC

### Operating System

Windows

### What happened

The alt text for the logo on the top left should be a human-readable string or be left out completely. It's counter-productive for screen readers.

![klipper-alt](https://user-images.githubusercontent.com/305679/211956619-298aad41-8baf-4b4a-bc99-39a4a0dd22c4.jpg)

### What did you expect to happen

Either to see a recognizable title or nothing at all.

### How to reproduce

Roll over the logo in top left corner with mouse.
